.github/workflows/zip-templates-and-add-to-release.yml

```diff
@@ -0,0 +1,33 @@
+name: zip-templates-and-add-to-release
+on:
+  release:
+    types: [published]
+jobs:
+  add-templates-to-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Install dependencies
+        run: |
+          npm install -g yarn
+          sudo apt-get update && sudo apt-get install -y --allow-unauthenticated zip
```

Comment:
This may not be needed as it looks like yarn and zip should already be installed for `ubuntu-latest` - https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md

Reply:
When running without either of these lines the relevant command fails because it's not installed. Im not sure why exactly if they are supposed to be installed in that version ubuntu. Probably a similar reason as the node comment above

Reply:
I have tried it without these installs and it works. Tested in my fork at https://github.com/ajmaddaford/design-system/runs/2390030841?check_suite_focus=true.

Reply:
Seems to work if I push changes. I think it wasn't working for me because I tested it using act. It fails if I run it with that